specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-08-01-preview/common-types.json

```diff
@@ -0,0 +1,81 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-03-01-preview",
```

Comment:
>"2020-03-01-preview", [](start = 15, length = 21)

Please update the version to be consistent with the API version

Reply:
fixed, thank you